WordPress/Classes/Models/WPAccount+Lookup.swift

```diff
@@ -0,0 +1,157 @@
+import CoreData
+
+public extension WPAccount {
+
+    // MARK: - Relationship Lookups
+
+    /// Is this `WPAccount` object the default WordPress.com account?
+    ///
+    @objc
+    var isDefaultWordPressComAccount: Bool {
+        guard let uuid = UserSettings.defaultDotComUUID, uuid.count > 0 else {
```

Comment:
I wonder if `!uuid.isEmpty` (or `uuid.isEmpty == false`) would be slightly clearer here than checking the count? It makes it more obvious that we're dealing with a string, perhaps (same for the method below too).

Reply:
Ya know... when I was reviewing I did a full stop to check that `count` was valid syntax there.  I'm still so used to strings having a length and arrays having a count 😆.